scripts/add-contribution.js

```diff
@@ -1,16 +1,15 @@
 const promptly = require('promptly');
 
-const ethers = require('ethers');
-const Kredits = require('../lib/kredits');
-
-const getNetworkId = require('./helpers/networkid.js')
+const initKredits = require('./helpers/init_kredits.js');
```

Comment:
❤️

Reply:
is it actually `init_kredits.js` or initKredits.js` as file name?

Reply:
I think JS folks use `require('./helpers/init-kredits');` You don't need the `.js`